src/pkg/services/m365/api/lists.go

```diff
@@ -595,7 +620,7 @@ func specifyODataType(filteredData map[string]any, colDetails *columnDetails, co
 	}
 
 	switch {
-	case colDetails.isPersonColumn:
+	case colDetails.isPersonColumn || colDetails.isLookupColumn:
```

Comment:
`,` should work too here. 

```suggestion
	case colDetails.isPersonColumn, colDetails.isLookupColumn:
```